ts/utils/paypal.ts

```diff
@@ -1,8 +1,34 @@
+import { NonNegativeNumber } from "@pagopa/ts-commons/lib/numbers";
 import { PayPalInfo } from "../../definitions/pagopa/PayPalInfo";
 import I18n from "../i18n";
+import { PspData } from "../../definitions/pagopa/PspData";
+import { IOPayPalPsp } from "../features/wallet/onboarding/paypal/types";
+import { PayPalPsp } from "../../definitions/pagopa/PayPalPsp";
+import { getPayPalPspIconUrl } from "./paymentMethod";
 
 // return the email used in the paypal account. If it can't be found "n/a" will be returned
 // it is inside the default psp ¯\_(ツ)_/¯ that could be one in the relative array 'pspInfo'
 export const getPaypalAccountEmail = (paypal: PayPalInfo): string =>
   paypal.pspInfo.find(p => p.default)?.email ??
   I18n.t("wallet.onboarding.paypal.emailNotAvailable");
+
+/**
+ * convert pspData (one of many PM representations of a psp) into IOPayPalPsp
+ * @param psp
+ */
+export const convertPspData = (psp: PspData): IOPayPalPsp => ({
+  id: psp.idPsp,
+  logoUrl: getPayPalPspIconUrl(psp.codiceAbi),
+  name: psp.ragioneSociale,
+  fee: psp.fee as NonNegativeNumber,
+  privacyUrl: psp.privacyUrl
+});
+
+// convert a paypal psp returned by the API into the app domain model
+export const convertPayPalPsp = (psp: PayPalPsp): IOPayPalPsp => ({
```

Comment:
🤓 **suggestion**: I would love to have one policy for these cross-domain functions.

Rather than having them scattered across `utils` files, would be great to have _transformers_ or _marshallers_ (you name it). This would encourage us to segregate the Client domain more and name things for what they are for the App rather than what they represent on the BE.

Reply:
Deal!
`utils` could be a bucket of everything
changed here 7e7c4a25df003c19017a889e19df2620ca299695